--- a/perl_inc_macro.h
+++ b/perl_inc_macro.h
@@ -37,7 +37,7 @@
 #endif
 
 #ifdef SITELIB_EXP
-#  if defined(WIN32)
+#  if defined(WIN32) && !defined(UNPIN_VFS_FIXED_INC)
     /* this picks up sitearch as well */
 #	  define INCPUSH_SITELIB_EXP s = PerlEnv_sitelib_path(PERL_FS_VERSION, &len); \
                 if (s) incpush_use_sep(s, len, INCPUSH_ADD_SUB_DIRS|INCPUSH_CAN_RELOCATE);
@@ -74,7 +74,7 @@
 #  define PRIVLIB_EXP "/usr/local/lib/perl5:/usr/local/lib/perl"
 #endif
 
-#if defined(WIN32)
+#if defined(WIN32) && !defined(UNPIN_VFS_FIXED_INC)
 #	define INCPUSH_PRIVLIB_EXP s = PerlEnv_lib_path(PERL_FS_VERSION, &len); \
     if (s) incpush_use_sep(s, len, INCPUSH_ADD_SUB_DIRS|INCPUSH_CAN_RELOCATE);
 #else
